app/ide-desktop/lib/client/src/index.ts

```diff
@@ -92,6 +95,19 @@ class App {
         return { ...configParser.parseArgs(argsToParse), fileToOpen, urlToOpen }
     }
 
+    setProjectToOpenOnStartup(idOfProjectToOpen: string) {
+        // Make sure that we are not initialized yet, as this method should be called before the
+        // application is ready.
+        if (!electron.app.isReady()) {
+            logger.log(`Setting project to open on startup: ${idOfProjectToOpen}.`)
+            this.args.groups.startup.options.project.value = idOfProjectToOpen
+        } else {
+            logger.error(
+                `Cannot set project to open on startup: ${idOfProjectToOpen}, as the application is already initialized.`
```

Comment:
line too long (121 chars). prettier *prefers* long lines in this case though, and will format it back if you split it across lines.

not sure whether this should just be added to the style guide as another exception

alternatively you can just change it to `'' + id + ''`